src/memory.rs

```diff
@@ -64,7 +71,7 @@ fn parse(contents: &str) -> HashMap<String, u32> {
         .collect()
 }
 
-pub fn get_memory_usage(config: &Config) -> String {
+pub fn get_usage(config: &MemoryConfig) -> String {
```

Comment:
i think some refactoring for this function is needed so we don't rely on any config here and instead fix the usage how we want it in the `print()` function below